index.html

```diff
@@ -358,130 +358,130 @@ <h4 class="functionality__paragraph">Necesito Ayuda</h4>
+                  <span class="most-popular__rating" aria-label="calificación de productos 4">4</span>
                 </div>
               </span>
             </div>
-          </div>
-        </div>
+          </li>
+        </ul>
         <div class="most-popular__right">
-          <a href="#" class="most-popular__link"></a>
+          <a href="#" class="most-popular__link" aria-label="ver Desayunador con 2 bancas color espresso"></a>
           <div class="most-popular__right-method">
             <div class="most-popular__right-shipping">
-              <img class="most-popular__right-close" src="image/Close.png" alt="Close">
-              <p class="Close">Envío a domicilio</p>
+              <img class="most-popular__right-close" src="image/Close.png" alt="">
+              <p class="Close" aria-label="envio a domicilio">Envío a domicilio</p>
             </div>
             <div class="most-popular__right-removal">
-              <img class="most-popular__right-check" src="image/Check.png" alt="Check">
-              <p class="Check">Retiro en tienda</p>
+              <img class="most-popular__right-check" src="image/Check.png" alt="">
+              <p class="Check" aria-label="Retiro en tienda">Retiro en tienda</p>
             </div>
           </div>
           <div class="most-popular__rigth-calification">
-            <span class="most-popular__rigth-star">&#9733;</span>
-            <span class="most-popular__rigth-rating">3</span>
+            <span class="most-popular__rigth-star" aria-hidden="true">&#9733;</span>
+            <span class="most-popular__rigth-rating" aria-label="calificación de productos 3">3</span>
           </div>
-          <img class="most-popular__right-img" src="image/Desktop-table.png" alt="">
+          <img class="most-popular__right-img" src="image/Desktop-table.png" alt="Desayunador con 2 bancas color espresso<">
           <div class="most-popular__right-text">
             <span class="most-popular__do">DO IT CENTER ONLINE </span>
             <span class="most-popular__right-name">Desayunador con 2 bancas color espresso</span>
             <div class="most-popular__right-total">
-              <p class="most-popular__right-price">209.99</p>
+              <p class="most-popular__right-price" aria-label="precio de producto $209.99">$209.99</p>
               <div class="most-popular__rigiht-entrega">
                 <img class="most-popular__prioridad" src="image/Entregahoy.png" alt="pioridades"> Entrega hoy
               </div>
             </div>
           </div>
           <div class="most-popular__bord"></div>
           <div class="most-popular__bord-botton">
-            <a href="#" class="most-popular__bord-button">VER PRODUCTO <img class="arrow" src="image/arrow-right.png" alt="">
+            <a href="#" class="most-popular__bord-button">VER PRODUCTO <img class="arrow" src="image/arrow-right.png" alt="Ver productos">
```

Comment:
Que productos? Dice `ver productos` pero, ¿que productos son? Ej:
```suggestion
            <a href="#" class="most-popular__bord-button">VER PRODUCTO <img class="arrow" src="image/arrow-right.png" alt="Ver todos los productos en promocion">
```